go/client/cmd_ctl_watchdog.go

```diff
@@ -6,129 +6,162 @@ package client
 import (
 	"fmt"
 	"os"
-	"syscall"
+	"path/filepath"
+	"runtime"
+	"sort"
+	"time"
 
 	"github.com/keybase/cli"
+	"github.com/keybase/client/go/install"
 	"github.com/keybase/client/go/libcmdline"
 	"github.com/keybase/client/go/libkb"
-	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-updater/watchdog"
 )
 
-const numRestartsDefault = 10
-
+// CmdWatchdog defines watchdog command
 type CmdWatchdog struct {
 	libkb.Contextified
-	restarts int
```

Comment:
git has rendered the diff here in an unfortunate way. i made minor changes to cmd_ctl_watchdog2, deleted cmd_ctl_watchdog, and renamed cmd_ctl_watchdog2 -> cmd_ctl_watchdog. these specific changes are almost certainly easier to review using the appropriately named commits.